coordinator/internal/authority/authority.go

```diff
@@ -1,29 +1,57 @@
 // Copyright 2024 Edgeless Systems GmbH
 // SPDX-License-Identifier: AGPL-3.0-only
 
+// Package authority guards the current state of the Coordinator.
```

Comment:
nit: Maybe we could rename to `stateguard`? It was called authority in the beginning because it issued the certs, not sure that's still its main function.

Reply:
Renamed